R/download_data_wrds_ccm_links.R

```diff
@@ -44,9 +44,10 @@ download_data_wrds_ccm_links <- function(
     filter(linktype %in% local(linktype) &
              linkprim %in% local(linkprim) &
              usedflag == local(usedflag)) |>
-    select(permno = lpermno, gvkey, linkdt, linkenddt) |>
+    select(permno = lpermno, gvkey,
+           start_date = linkdt, end_date = linkenddt) |>
```

Comment:
@christophscheuch I suggest "link_" prefix, otherwise, I think it is a bit meaningless.